pkg/asndb/asndb.go

```diff
@@ -0,0 +1,65 @@
+package asndb
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+const url string = "http://www.cidr-report.org/as2.0/asn.txt"
+const asnFormat string = `([\d]+)\s+(.*),\s(\w{2})`
```

Comment:
Rather than have these as constants, would instead make sense to have these as attributes on a struct, with the functions below defined defined on that struct (as methods)?

Reply:
Updated in 63d9e55.